src/style.css

```diff
@@ -1,3 +1,65 @@
+html {
+  --main-color: #00ACDC;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  min-width: 1024px;
+}
+
+header {
+  width: calc(100% - 100px);
```

Comment:
redundant - this property doesn't make any effect now

Reply:
Hi! Thank you for your feedback! I hope I have corrected all the necessary things :)